.idea/misc.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please remove all OS generated files from the repo.  you can use ``git rm <filename>`` to remove them locally but also from the repo.  Then it is a good idea to modify your local ``.gitignore`` file to ignore these in future.  We could also add it to the project ``.gitignore`` file,  but it is a better fix for you to add it to your defaut local one so that these files are not added to any of your projects.